server/ladder_service.py

```diff
@@ -19,14 +23,57 @@ class LadderService:
     Service responsible for managing the 1v1 ladder. Does matchmaking, updates statistics, and
     launches the games.
     """
-    def __init__(self, games_service: "GameService"):
+    def __init__(self, games_service: GameService):
         self._informed_players: Set[Player] = set()
         self.game_service = games_service
 
+        # Hardcoded here until it needs to be dynamic
```

Comment:
No
Should be made dynamic in the first place
In fact, I think Ladder1v1 and LadderXvX should be exactly the same and using the same code (1v1 is no special case)
Let's think the "2v2" as "XvX" right now so that we can easily expand it in the future

Reply:
Dynamic means "happens at runtime." So dynamically adding queues would mean something like adding a control server endpoint to configure the queues without needing to restart the server. This is (IMO), to quote yourself, "massively overkill" and something that is not necessary for team matchmaking MVP.

I agree that in theory Ladder1v1 feels like it should be exactly the same as XvX. However, ladder1v1 requires that the game mode be set to `ladder1v1` which instantiates the game object as a `LadderGame` instead of a `CustomGame`, that the rating be using `ladder_rating`, and currently the game configures both ladder players to be in the FFA(-) team, which obviously won't work for XvX.

Basically, it means in order to combine ladder1v1 with ladderXvX, we will have to change how ladder1v1 games behave, and I didn't want to do that yet since that isn't the point of this PR.